src/plugins/saved_objects/public/save_modal/saved_object_save_modal.tsx

```diff
@@ -101,80 +99,71 @@ export class SavedObjectSaveModal extends React.Component<Props, SaveModalState>
-                    aria-describedby={this.state.hasTitleDuplicate ? duplicateWarningId : undefined}
-                  />
-                </EuiFormRow>
-
-                {this.renderViewDescription()}
-
-                {typeof this.props.options === 'function'
-                  ? this.props.options(this.state)
-                  : this.props.options}
-              </EuiForm>
-            </EuiModalBody>
-
-            <EuiModalFooter>
-              <EuiButtonEmpty data-test-subj="saveCancelButton" onClick={this.props.onClose}>
-                <FormattedMessage
-                  id="savedObjects.saveModal.cancelButtonLabel"
-                  defaultMessage="Cancel"
-                />
-              </EuiButtonEmpty>
-
-              {this.renderConfirmButton()}
-            </EuiModalFooter>
-          </EuiModal>
-        </form>
-      </EuiOverlayMask>
+      <EuiModal
+        data-test-subj="savedObjectSaveModal"
+        className="osdSavedObjectSaveModal"
+        onClose={this.props.onClose}
+      >
+        <EuiModalHeader>
+          <EuiModalHeaderTitle>
+            <FormattedMessage
+              id="savedObjects.saveModal.saveTitle"
+              defaultMessage="Save {objectType}"
+              values={{ objectType: this.props.objectType }}
+            />
+          </EuiModalHeaderTitle>
+        </EuiModalHeader>
+
+        <EuiModalBody>
+          {this.renderDuplicateTitleCallout(duplicateWarningId)}
+
+          <EuiForm component="form" onSubmit={this.onFormSubmit} id="savedObjectSaveModalForm">
+            {!this.props.showDescription && this.props.description && (
+              <EuiText size="s" color="subdued">
+                {this.props.description}
+              </EuiText>
+            )}
+
+            <EuiSpacer />
+
+            {this.renderCopyOnSave()}
+
+            <EuiFormRow
+              fullWidth
+              label={
+                <FormattedMessage id="savedObjects.saveModal.titleLabel" defaultMessage="Title" />
+              }
+            >
+              <EuiFieldText
+                fullWidth
+                autoFocus
+                data-test-subj="savedObjectTitle"
+                value={title}
+                onChange={this.onTitleChange}
+                isInvalid={(!isTitleDuplicateConfirmed && hasTitleDuplicate) || title.length === 0}
+                aria-describedby={this.state.hasTitleDuplicate ? duplicateWarningId : undefined}
+              />
+            </EuiFormRow>
+
+            {this.renderViewDescription()}
+
+            {typeof this.props.options === 'function'
+              ? this.props.options(this.state)
+              : this.props.options}
+          </EuiForm>
+        </EuiModalBody>
+
+        <EuiModalFooter>
+          <EuiButtonEmpty data-test-subj="saveCancelButton" onClick={this.props.onClose}>
+            <FormattedMessage
+              id="savedObjects.saveModal.cancelButtonLabel"
+              defaultMessage="Cancel 2"
```

Comment:
Why `2`?

Reply:
lol. forgot to remove this. thanks for catching this.

Reply:
Fixed.